### DISCUSSION TITLE
how to enable everything...

### DISCUSSION THREAD

**Discussion by Blazzycrafter** [Flow-Launcher/Flow.Launcher#1936] [Q&A] [answered]
I used everything and completely disabled win search service…
I did not find the option to use everything instead of win search index…

**Comment by jjw24** [comment, marked as answer]
If by disabling you mean disabling the actual windows service, you don't need to do that. Just go to go the Explorer plugin settings and select the search engine as Everything. Start typing and you will get a prompt to start/install your Everything app. After doing so you will be able to search with everything via flow.

**Reply by jjw24** [reply]
FYI Everything 1.5a is not fully supported yet but there is a workaround. Otherwise I recommend to get the prior version of Everything.

**Reply by Blazzycrafter** [reply]
yeah i disabled the complete windows search service...
windows meas to start a index if i play physic intense games...
XD

Thanks btw
